packages/ketcher-react/src/script/ui/views/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,126 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+import React, { useRef, useEffect } from 'react'
+import * as KN from 'w3c-keyname'
+import clsx from 'clsx'
+
+import style from './Dialog.module.less'
+
+interface DialogProps {
+  children: React.ReactElement
+  title: string
+  params: {
+    className: string
+    onCancel: () => void
+    onOk: (result: any) => void
+  }
+  result: () => any
+  valid: () => boolean
+  buttons: Array<string | React.ReactElement>
+  className: string
+}
```

Comment:
please use Props and CallProps as above

Reply:
2179bc2582e70c743bc411e6970a6b5ba7b4f8c4